gem_controllers/stages/operation_point_selection/scim_ops.py

```diff
@@ -0,0 +1,203 @@
+import gym_electric_motor as gem
+import numpy as np
+import scipy.interpolate as sp_interpolate
+import matplotlib.pyplot as plt
+from .operation_point_selection import OperationPointSelection
+from ..base_controllers import PIController
+from ..anti_windup import AntiWindup
+
+
+class SCIMOperationPointSelection(OperationPointSelection):
```

Comment:
This class looks quite similar to the PMSM Operation Point selection class.

Wouldn't it be beneficial to use a common base class (e.g. FieldOrientedControllerOperationPointSelection)?